.changes/1990-event-list-improvements.md

```diff
@@ -0,0 +1,4 @@
+- [New] : Better visualisation of Ongoing, Upcoming and Past events by colours
+- [New] : Event listing screen contains Search and Filters option for better explore your events
+- [Fix] : Now dashboard will show events which you are attending
+- [Improvement] : General code improvements and bugs fixing in event listing
```

Comment:
I don't think this is probably wanted to be relevant for end-users, unless you want to re write mentioning the specific bug(s) which is fixed with issue it was causing...

Reply:
Regarding mentioning specific issue, there are many of the improvements/bugs which are not part of current issue board but fixed as side things in this type of refactoring stuff.

In reference to end user, except last point, all other 3 points are important I believe.

Still open for any suggestion from you or @gnunicorn with regards to this.

Reply:
Oh I did meant for the last one only! Sorry for not explicit mention of it in my wording...